### PR TITLE
Vendor in latest golang.org/x/sys

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@nalind PTAL

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by rhatdan** [comment]
@nalind Can I merge this in?

File: vendor.conf [hunk 1]
```diff
@@ -17,4 +17,7 @@ github.com/tchap/go-patricia v2.2.6
 github.com/vbatts/tar-split bd4c5d64c3e9297f410025a3b1bd0c58f659e721
 github.com/vdemeester/shakers 24d7f1d6a71aa5d9cbe7390e4afb66b7eef9e1b3
```
Comment (nalind) [inline vendor.conf]: This PR removes the module and `github.com/go-check/check`.  Remove them while we're in here?

File: vendor.conf [hunk 2]
```diff
@@ -17,4 +17,7 @@ github.com/tchap/go-patricia v2.2.6
...
 golang.org/x/net f2499483f923065a842d38eb4c7f1927e6fc6e6d
-golang.org/x/sys d75a52659825e75fff6158388dddc6a5b04f9ba5
+golang.org/x/sys 07c182904dbd53199946ba614a412c61d3c548f5
+github.com/stretchr/testify 4d4bfba8f1d1027c4fdbe371823030df51419987
+github.com/davecgh/go-spew 346938d642f2ec3594ed81d874461961cd0faa76
+github.com/pmezard/go-difflib v1.0.0
```
Comment (nalind) [inline vendor.conf]: This list is sorted, please keep it that way.

**Comment by nalind** [comment]
A pair of nits in `vendor.conf`, otherwise LGTM.

**Comment by rhatdan** [comment]
Ok updated vendor.conf, will merge when the travis passes.

**Comment by nalind** [comment]
[Backport overlayxfs quota support from moby to cri-o](https://trello.com/c/wonuFj8Z/666-backport-overlayxfs-quota-support-from-moby-to-cri-o)

**Comment by nalind** [comment]
[(8) Containers/storage needs to back port moby/moby support for quota support.](https://trello.com/c/5E3mkECi/679-8-containers-storage-needs-to-back-port-moby-moby-support-for-quota-support)

**Pull request merged**